package.json

```diff
@@ -52,6 +52,7 @@
     "ldapjs": "1.0.1",
     "lodash": "4.17.4",
     "moment": "2.18.1",
+    "package-json": "4.0.1",
```

Comment:
@xPaw, could you remove your commit out of this PR? This PR is essentially ready, and I'm expecting it to be merged during the weekend.